release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1030,6 +1030,10 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-cloud-compute-bug-fixes_{context}"]
 ==== Cloud Compute
 
+* Previously, when an {product-title} cluster was installed with no capabilities and later enabled the Build capability, the related Build cluster configuration custom resource definition (CRD) was not created. With this release, the Build cluster configuration CRD and its default instance are created. This allows the Build capability to be fully configured and customized. (link:https://issues.redhat.com/browse/OCPBUGS-34395[*OCPBUGS-34395*])
+
+* Previously, role bindings related to the Image Registry, Build, and DeploymentConfig capabilities were created in every namespace, even if the respective capability was disabled. With this release, role bindings will only be created if the respective capability is enabled on the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-34077[*OCPBUGS-34077*])
```

Comment:
```suggestion
* Previously, role bindings related to the Image Registry, Build, and DeploymentConfig capabilities were created in every namespace, even if their capability was disabled. With this release, role bindings will only be created if their capability is enabled on the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-34077[*OCPBUGS-34077*])
```
If possible, rewrite to avoid using 'respective'. See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r

Example above.